source/read-write-concern/tests/operation/default-write-concern-3.4.yml

```diff
@@ -0,0 +1,92 @@
+# Test that setting a default write concern does not add a write concern
+# to the command sent over the wire.
+# Test operations that require 3.4+ server, where all commands started
+# to accept a write concern.
+
+data:
+  - {_id: 1, x: 11}
+  - {_id: 2, x: 22}
+collection_name: &collection_name default_write_concern_coll
+database_name: &database_name default_write_concern_db
+other_collection_name: &other_collection_name other_collection_name
+
+runOn:
+    - minServerVersion: "3.4"
+
+tests:
+  - description: Aggregate with $out omits default write concern
+    operations:
+      - object: collection
+        collectionOptions: {writeConcern: {}}
+        name: aggregate
+        arguments:
+          pipeline: &out_pipeline
+            - $match: {_id: {$gt: 1}}
+            - $out: *other_collection_name
+    outcome:
+      collection:
+        name: *other_collection_name
+        data:
+          - {_id: 2, x: 22}
+    expectations:
+      - command_started_event:
+          command:
+            aggregate: *collection_name
+            pipeline: *out_pipeline
+            writeConcern: null
+  - description: RunCommand with a write command omits default write concern
```

Comment:
This test, while correct, is a little misleading. The generic runCommand is never supposed to add writeConcern so it should omit non-default writeConcerns too: https://github.com/mongodb/specifications/blob/master/source/read-write-concern/read-write-concern.rst#id6
> If your driver offers a generic RunCommand method on your database object, WriteConcern MUST NOT be applied automatically to any command. A user wishing to use a WriteConcern in a generic command must manually include it in the command document passed to the method.

Also (assuming we keep this test) can we use a drop command instead of create? That will ensure that the test can be run multiple times without needing to drop the database.

Reply:
Good point. Changed to a `drop` command and added to the description to note that runCommand should never inherit a write concern.

Reply:
Good catch, @ShaneHarvey!

Reply:
Whoops, I just learned that dropping a non-existent collection will fail with "ns not found". Can we change this to run a run-op delete command (or something else that can be run multiple times in a row)? Sorry for the churn.

Reply:
Updated. Oddly, libmongoc's spec test runner actually ignored the ns not found error. I think that might be a bug on our end.